src/lib/PreviewUI.js

```diff
@@ -109,7 +113,8 @@ class PreviewUI {
         insertTemplate(this.container, shellTemplate);
 
         this.container = this.container.querySelector(SELECTOR_BOX_PREVIEW_CONTAINER);
-        this.contentContainer = this.container.querySelector(SELECTOR_BOX_PREVIEW);
```

Comment:
Do any other classes extend PreviewUI and use this property? Do they work properly, if so?

Reply:
LOL yes referenced by `DocPreloader.checkDocumentLoaded` good catch!